src/types.ts

```diff
@@ -159,6 +159,18 @@ export enum ModuleName {
   EtherDividendCheckpoint = 'EtherDividendCheckpoint',
 }
 
+export enum TransferStatusCodes {
```

Comment:
You can encode the status codes in the enum itself, like

```
export enum TransferStatusCode {
  TransferFailure = '0x50',
  TransferSuccess = '0x51',
  //  ...etc
}
```

And then use them for the switch statement above instead of comparing with the actual string value

Also, the enum name should be singular, not plural